README.rst

```diff
@@ -97,6 +97,25 @@ patched and that you don't have to worry about the details.
 API
 ===
 
+``replace(func, new_source)``
+-----------------------------
+
+Replace the source code of function ``func`` with ``new_source``.
```

Comment:
+ "This is highly unrecommended as it means if the original function changes, the call to `replace()` will continue to silently succeed."